### ISSUE TITLE
v7 -> v8 state error in stock/post-migration.py

### ISSUE THREAD

**Issue by bizslink-calon** [OCA/OpenUpgrade#1783]
```
def migrate_stock_qty(cr, registry):
    """Reprocess stock moves in done state to fill stock.quant."""
    # First set restrict_lot_id so that quants point to correct moves
    with api.Environment.manage():
        env = api.Environment(cr, SUPERUSER_ID, {})
        moves = env['stock.move'].search(
            [('state', 'in', ['assign', 'done'])], order="date")
        for move in moves:
            if move.state == 'assign':
                _move_assign(env, move)
            else:
                _move_done(env, move)
```

error state 'assign', maybe assigned?

**Comment by pedrobaeza** [comment]
Which is the error?

**Comment by bizslink-calon** [comment]
> Which is the error?

'assign' is not in stock_move.state's selection instead of 'assigned'.

**Comment by pedrobaeza** [comment]
You're totally right! Fixed in 8cd47363f42cd03e910caca03bf16ebbee45ee75

**Comment by bizslink-calon** [comment]
> You're totally right! Fixed in [8cd4736](https://github.com/OCA/OpenUpgrade/commit/8cd47363f42cd03e910caca03bf16ebbee45ee75)

You may rewrite this method, somewhere still used 'assign'.

**Comment by pedrobaeza** [comment]
OK, the oversight has been fixed in 6f465f34e91940b3075fb2c1cf99b3a8b28677da
